Stripe3DS2/Stripe3DS2/Resources/Stripe3DS2.xcassets/amex-logo.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "american-express@1x.pdf"
```

Comment:
I'm not seeing this as an asset in `master` was this a mistake?  pdf?

Reply:
This is the original vector source file for the logo, Xcode will split it into PNGs at compile time.